models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -466,13 +464,20 @@ aeif_cond_beta_multisynapse::calibrate()
 
   for ( size_t i = 0; i < P_.num_of_receptors_; ++i )
   {
-    double t_p = P_.taus_decay[ i ] * P_.taus_rise[ i ]
-      / ( P_.taus_decay[ i ] - P_.taus_rise[ i ] )
-      * std::log( P_.taus_decay[ i ] / P_.taus_rise[ i ] ); // peak time
-    V_.g0_ex_[ i ] = V_.g0_in_[ i ] // normalization factors for conductance
-      = ( 1. / P_.taus_rise[ i ] - 1. / P_.taus_decay[ i ] )
-      / ( std::exp( -t_p / P_.taus_decay[ i ] )
-          - std::exp( -t_p / P_.taus_rise[ i ] ) );
+    if ( P_.taus_decay[ i ] == P_.taus_rise[ i ] ) // alpha function limit
+    { // use normalization for alpha function in this case
+      V_.g0_ex_[ i ] = V_.g0_in_[ i ] = 1.0 * numerics::e / P_.taus_decay[ i ];
```

Comment:
I would suggest to avoid statements with serial assignments, there is always a risk that one overlooks the "sandwiched" assignment, especially if the expression to the right is not totally trivial (eg `a = b = c = 0`).